src/framework/utils.hpp

```diff
@@ -335,7 +335,143 @@ cmatrix_t Matrix::U3(double theta, double phi, double lambda) {
   return mat;
 }
 
+//------------------------------------------------------------------------------
+// Static Vectorized Matrices
+//------------------------------------------------------------------------------
+class VMatrix {
+public:
+  // Single-qubit gates
+  const static cvector_t I;     // name: "id"
+  const static cvector_t X;     // name: "x"
+  const static cvector_t Y;     // name: "y"
+  const static cvector_t Z;     // name: "z"
+  const static cvector_t H;     // name: "h"
+  const static cvector_t S;     // name: "s"
+  const static cvector_t Sdg;   // name: "sdg"
```

Comment:
To keep consistency with the rest of constant names, I'd rename it to: `SDG`

Reply:
Member names of `AER::Utils::VMatrix` follow `AER::Utils::Matrix`. I will fix the both.